src/utils/exportUtils/HtmlExport.tsx

```diff
@@ -96,9 +96,9 @@ export default class HTMLExporter extends Exporter {
         const roomAvatar = await this.getRoomAvatar();
         const exportDate = formatFullDateNoDayNoTime(new Date());
         const creator = this.room.currentState.getStateEvents(EventType.RoomCreate, "")?.getSender();
-        const creatorName = this.room?.getMember(creator)?.rawDisplayName || creator;
-        const exporter = this.client.getUserId();
-        const exporterName = this.room?.getMember(exporter)?.rawDisplayName;
+        const creatorName = this.room.getMember(creator!)?.rawDisplayName || creator;
```

Comment:
```suggestion
        const creatorName = creator ? this.room.getMember(creator)?.rawDisplayName : creator || creator;
```

Reply:
It seems that SonarCloud [does not prefer this formulation](https://sonarcloud.io/project/issues?resolved=false&pullRequest=9975&id=matrix-react-sdk&open=AYXkGSk8ltR4PlskDqM4&tab=code).

Reply:
Revert it then :)

Reply:
Looks like it was a precedence issue. I guess it was interpreted as:
```typescript
creator ? this.room.getMember(creator)?.rawDisplayName : (creator || creator);
```
which is actually a subtle bug. Today I learned!